components/layout/AppHeader/AppHeader.jsx

```diff
@@ -1,20 +1,33 @@
-import { AppBar, makeStyles, Toolbar } from '@material-ui/core';
+import { AppBar, Box, makeStyles, Toolbar, Typography } from '@material-ui/core';
 import React from 'react';
 import Logo from '../../common/Logo/Logo';
+import { JoinUsHeaderButton } from '../JoinUsHeader/JoinUsHeaderButton';
 
-const useStyles = makeStyles(() => ({
-  root: {
-    backgroundColor: 'orangered',
+const useStyles = makeStyles(({ spacing }) => ({
+  menu: {
+    flexGrow: 1,
+    display: 'flex',
+    flexDirection: 'row',
+    '& >*': {
+      padding: spacing(0, 1),
+    },
   },
 }));
 
 export const AppHeader = () => {
   const classes = useStyles();
 
   return (
-    <AppBar position="static" className={classes.root}>
+    <AppBar position="static" color="transparent" elevation={0}>
       <Toolbar>
         <Logo />
+        <Box m={1} className={classes.menu}>
```

Comment:
Tohle není Mobile-first.

Určitě bych kodil raději Mobile-first. A doporučuje to u MaterialUI

Reply:
upraveno pro mobile-first.